src/c_api/c_api.cc

```diff
@@ -1573,6 +1578,42 @@ int MXRandomSeedContext(int seed, int dev_type, int dev_id) {
   API_END();
 }
 
+int MXSetFlushDenorms(bool value) {
+  API_BEGIN();
+  // FTZ only applies to SSE and AVX instructions.
+  #if defined(__SSE__) || defined(_M_X64) || (defined(_M_IX86_FP) && _M_IX86_FP >= 1)
+    auto is_dmz_flag_available = []() {
```

Comment:
To be very picky here, I'd rather prefer to see a bool type instead of auto type. Well. If the lambda's body is a single return statement, then the return type might be defined explicitly,